src/components/Filters/components/Input/Input.tsx

```diff
@@ -44,25 +49,37 @@ export const StyledInput = styled.input`
   :-ms-input-placeholder {
     color: ${getFormStyle('placeholderColor')};
   }
+
+  ${({ isInvalid }) =>
+    isInvalid &&
+    css`
+      ${stateStyles}
+      border-color: ${getFormStyle('invalidBorderColor')};
+    `}
 `;
 
 const Input: React.FC<InputProps> = ({
   value = '',
   onChange,
   maxLength,
-  pattern,
-  patternMessage,
+  pattern = patterns.string.pattern,
+  patternMessage = patterns.string.patternMessage,
+  validate = validateDefault,
```

Comment:
[minor] Prop should be named `onValidate`. `validateDefault` then evaluate the pattern, right? I would name it `patternValidation` or something more descriptive